### ISSUE TITLE
NTR: Phlebolithiasis (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8994]
New term request Phlebolithiasis (segal_030319185045): Calcified thrombi in veins  UMLS: 0031547

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000661
- label: Phlebolith
- parents: [Abnormal thrombosis (HP:0001977)](https://hpo.jax.org/app/browse/term/HP:0001977)
- pmid: PMID:37981851
- def: A calcified structure that forms within a vein. The pathogenesis of phleboliths is thought to involve an organized thrombus produced when the peripheral blood flow slows. The thrombus calcifies, form- ing the core of the phlebolith. Then, the fibrinous com- ponent undergoes secondary calcification and becomes attached. Repetition of this process causes enlargement of the phlebolith.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8994
- synonyms:
